gapis/api/vulkan/command_buffer_rebuilder.go

```diff
@@ -921,6 +921,9 @@ func rebuildVkCmdExecuteCommands(
 		if !GetState(s).CommandBuffers().Contains(buf) {
 			return nil, nil, fmt.Errorf("Cannot find CommandBuffer %v", buf)
 		}
+		if GetState(s).CommandBuffers().Get(buf).Recording() != RecordingState_COMPLETED {
```

Comment:
To reviewers: I guess a question is: is it OK to keep on generating the replay if we detect such an issue, or should we abort the replay entirely?

In the test trace I used to debug this (from a Unity demo), the replay works fine with this fix.

Reply:
After a chat with @AWoloszyn , he agreed this is a relevant fix: it's OK to not recreate the primary commandbuffer, which would be invalid to submit anyway.